examples/tutorials/Differentiation_Infrastructure_in_Deepchem.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<p>Please add explanation of code as comments.</p><p>Please print some text describing the outputs, eg. </p><p>"Root 1: [-3.124]</p><p>Root 2: [-9.657]"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Seconding. Also, please add a section at top about the mathematical problem of rootfinding. You need to explain why people should care about rootfinding as a mathematical operation</p>